scripts/teardown.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Should we add `set -e` right after the shebang? (Fail fast if any command fails)

---

ah, wasn't aware of that command

let me read up on that and add